middle_end/flambda2/terms/code_size.ml

```diff
@@ -385,6 +386,7 @@ let ternary_prim_size prim =
   match (prim : Flambda_primitive.ternary_primitive) with
   | Block_set (block_access, init) -> block_set block_access init
   | Array_set kind -> array_set kind
+  | Array_vector_set (_vec, kind) -> array_set kind
```

Comment:
same remark here, the actual instructions for 128-bits might be bigger than the ones regularly used for arrays.

Reply:
(Same remark)